.travis.yml

```diff
@@ -79,10 +72,9 @@ jobs:
         - npm run build
 
     - stage: test
-      <<: *stage_test_go
```

Comment:
Travis tries to run some ruby - I wonder if this line should go back in? But I got to admit that @tharvik script-fooed the hell out of .travis.yml and that I never know where a `-` is needed and where not.

Reply:
`<<:` is the merge operator, taking its definition and merging it with the current dict.
```yaml
- stage: test
<<: *stage_test_go
---
- stage: test
  before_script: *get_coding
  script: GO111MODULE=on make test_goveralls
```
```yaml
- stage: test
- <<: *stage_test_go
---
- stage: test
- before_script: *get_coding
  script: GO111MODULE=on make test_goveralls
```